publication-rest/src/main/java/no/unit/nva/publication/uploadfile/CompleteUploadHandler.java

```diff
@@ -0,0 +1,53 @@
+package no.unit.nva.publication.uploadfile;
```

Comment:
Placing these handler in publication-rest.uploadfile as I see download file handlers are also placed here. 
Should consider own file module?

Reply:
YES!